tree/ntuple/v7/src/RDaos.cxx

```diff
@@ -210,21 +210,22 @@ int ROOT::Experimental::Detail::RDaosContainer::ReadSingleAkey(void *buffer, std
                                                                DistributionKey_t dkey, AttributeKey_t akey,
                                                                ObjClassId_t cid)
 {
-   AttributeKey_t akeys[] = {akey};
-   d_iov_t iovs[1];
-   d_iov_set(&iovs[0], buffer, length);
-   RDaosObject::FetchUpdateArgs args(dkey, akeys, iovs);
+   std::vector<d_iov_t> iovs(1);
+   d_iov_set(&iovs[0], const_cast<void *>(buffer), length);
+   RDaosObject::RDaosRequest requests[] = {{akey, iovs}};
```

Comment:
See comment on the constructor of `RDaosObject::RDaosRequest`.  Maybe we can avoid allocating a vector and just using a single `d_iov_t` allocated in the stack.

Otherwise, let's ensure that `iovs` is not copied, i.e.
```suggestion
   RDaosObject::RDaosRequest requests[] = {{akey, std::move(iovs)}};
```